app/assets/javascripts/discourse/tests/acceptance/user-test.js

```diff
@@ -177,3 +179,123 @@ acceptance("User - Saving user options", function (needs) {
     );
   });
 });
+
+acceptance(
+  "User - Notification level when `can_{mute|ignore}_user` are false",
```

Comment:
```suggestion
  "User - Notification level dropdown visibility",
```

Goes with suggestion below